options_compile.cpp

```diff
@@ -192,6 +194,49 @@ std::string EffectiveOptionsFilter::processOptions(const OpenCLArgList &args,
 
   effectiveArgs.push_back(szTriple);
 
+  for (OpenCLArgList::const_iterator it = argsEx.begin(), ie = argsEx.end();
```

Comment:
Can we use 
```
fp64Enable = argsEx.hasArgNoClaim(OPT_COMPILE_cl_khr_fp64);
```
instead?

Reply:
According to my local test, cl-ext=...,... info is contained in pszOptionsEx, so I added check for args in pszOptionsEx.

Reply:
I've updated the patch as request.